src/Symfony/Component/HttpKernel/Tests/DependencyInjection/RegisterControllerArgumentLocatorsPassTest.php

```diff
@@ -236,7 +236,10 @@ public function testNoExceptionOnNonExistentTypeHintOptionalArg()
         $pass->process($container);
 
         $locator = $container->getDefinition((string) $resolver->getArgument(0))->getArgument(0);
-        $this->assertSame(['foo::barAction', 'foo::fooAction'], array_keys($locator));
+
+        $services = array_keys($locator);
+        sort($services);
```

Comment:
you could use `assertEqualsCanonicalizing()` instead of sorting yourselves

Reply:
@stof thank you